### DISCUSSION TITLE
Modificación de la ficha de revisión proyectos

### DISCUSSION THREAD

**Discussion by Daninja111** [ISPC-TST-Electronica-Microcontrolada/Grupo4#52] [Show and tell]
Chicos! 🤖 agregué cambios a la ficha de revisión, les parece bien actualmente? Algo que quieran agregar? Adjunto texto ampliado:
![image](https://user-images.githubusercontent.com/89562060/189502945-5970bed0-540c-497c-b30d-4e423c3a0468.png)

@guelo2019 @matiluajn @alfredop37 @maxii-sc @ivanexegomez

**Comment by guelo2019** [comment]
Hola, para mi esta perfecto!

**Comment by matiluajn** [comment]
Excelente dani! Buen trabajo.
